### ISSUE TITLE
Requirements

### ISSUE THREAD

**Issue by HumpyBumpy** [beaston02/ChaturbateRecorder#30]
Can this script be ran from a Raspberry Pi?
What are the requirements and memory usage of the script?
Thx!

**Comment by pokbb** [comment]
@HumpyBumpy 

Yes, it depends the amount of models that you are recording. With 7/10 models you need about 20/40% of cpu and 200/300mb of ram, but you need restart the script every 24/72 hours because the script cosume more ram that it need and can freeze the pi.

I use this script and two more with no problem.
Regards.

**Comment by beaston02** [comment]
As stated above, it will run just fine on a raspberry pi. The script definitely isn't perfect. There are some things I haven't been able to figure out which does continue to use more resources as time goes (as far as I can to, it's a bug with some modules being used, but I also still consider myself a beginner so very possible it's my lack of knowledge). 

Something you could do is run a cron job to restart the script every so often if models aren't being recorded. I've thought about adding this as an option to the script also to help keep its demands on the system down. I've been strapped on time and haven't done any work on it (or the other scripts) lately because of that.
